frontend/src/components/funding/FundingAgreementsTab.vue

```diff
@@ -59,12 +61,42 @@ export default {
     ...mapState(useAuthStore, ['userInfo']),
   },
 
-  created() {
+  async created() {
+    this.loading = true
     this.format = format
     this.FUNDING_AGREEMENT_STATUS_CODES = FUNDING_AGREEMENT_STATUS_CODES
+    await this.loadApprovedSuppApps()
+    this.loading = false
   },
 
   methods: {
+    async loadApprovedSuppApps() {
+      try {
+        this.loading = true
+        const applications = await ApplicationService.getActiveApplications()
+
+        this.supplementaryApplications = (
+          await Promise.all(
+            applications.map((application) => ApplicationService.getSupplementaryApplications(application.applicationId, `statusCode=${SUPPLEMENTARY_APPLICATION_STATUS_CODES.APPROVED}`)),
```

Comment:
There is a search card in this screen, and I don't think it works well with your Supp Funding.

Reply:
should work better now !